### ISSUE TITLE
What is happiness?

### ISSUE THREAD

**Issue by franciscolourenco** [nicksellen/ponderings#8]

**Comment by nicksellen** [comment]
Yes, it's an interesting point you make that I hadn't considered. Do you think it is realistic though?

**Comment by franciscolourenco** [comment]
Not really, but who knows.

**Comment by serapath** [comment]
It's pretty important :-) Most people say - I just go with what "feels right" or "what makes me happy".
- Might sometimes be an illusion (thus it doesn't make happy, but people do not realize it).
- Sometimes people find happiness in stuff that make things worse for others without or with realizing it

I wonder if everything has the potential to make people happy or at least if there is a broad range of things that can make people equally happy, then is there a way to decide what or how to make myself happy?

Or do we not have any power over what makes us happy, because it's kind of "baked into us" and we are now stuck with whatever our guts tell us? Which might mean, its not even worth discussing, right?

**Comment by nicksellen** [comment]
@aristidesfl I was being silly as there was just a blank comment at the top. I was somehow imaging I had already written _something_ about what happiness _could be_ on https://github.com/nicksellen/ponderings/blob/master/happiness.md. but not really yet.

@serapath I think it's worth discussing, even if it doesn't come to a solid answer.

I think the thing on https://github.com/nicksellen/ponderings/blob/master/happiness.md adds one potentially useful split, the difference between _contentment_ and _happiness_ (contentment is more like your estimation of how your life is doing and happiness is more about positive emotions recently).

**Comment by franciscolourenco** [comment]
@nicksellen I was going along :)

It is interesting that most answers to this question, end up digressing on how to get happiness, how to be happy, different types of happiness, such is our obsession with this magical thing. But what is it actually?

**Comment by nicksellen** [comment]
It's hard to know, the wikipedia page includes

> Philosophers and religious thinkers often define happiness in terms of living a good life, or flourishing, rather than simply as an emotion.

And I think this what I am getting at when I say happiness. I don't mean a single moment of happiness, I mean lasting happiness. So it might need to be split out into:
- the immediate sensation of happiness
- living a life that includes enough of these moments, and not too many negative ones

I don't know what `enough` or `too many` would be but that doesn't necessarily matter, it's possible to define a concept without being able to measure it.

**Comment by serapath** [comment]
honestly, turn it however you want, it basically boils down to having access to ressources (this mean money). Once you have that, the only thing left are "luxury problems" which is what you get when you are confronted with too many options and cannot decide what to do.
I dont know how to solve that, but other people and their needs is a good indicator on what you could do with your time and happyness in my experience comes from being with others and making them happy.

**Comment by aristidesfl** [comment]
Or so believe the people who are not wealthy. 

Once you get what you currently think you need to be happy, the hedonic treadmill doesn't stop. That is one thing wealthy people know that poor don't.

Maybe we should open a new issue to talk about how to be happy, cause this topic distracts from the main question.

**Comment by serapath** [comment]
lol, thats a quite arrogant and ignorant statement

**Comment by serapath** [comment]
As long as you get in money every month, so that you can pay rent of a cheap small appartement and you can go to the supermarket and buy the food you need to stay healthy and you have some freedom and don't need to work 60+ hours under whatever conditions... which is pretty humble requirements, then you can say "luxury starts" and yes maybe, the hedonic treadmill never stops...

...but honestly, if you all the time have to fight with existential fears that you might end up on the street hungry fighting your way through just to survive and you have people telling that money doesnt matter ... while they go and buy new food or clothing and go to take a shower in a proper apartement... ...then that has kind of a really bitter taste.

**Comment by nicksellen** [comment]
> it basically boils down to having access to ressources (this mean money)

seems to directly contradict

> happyness in my experience comes from being with others and making them happy

There is a common idea that having money "up to a point" makes you happier, then after that it doesn't. I have a study that supports that view on the [future page](https://github.com/nicksellen/ponderings/blob/master/happiness.md).

But @aristidesfl is right, this is all a different topic, the question is what _is_ happiness, not how to achieve it.

**Comment by serapath** [comment]
### It does not contradict.

**Try to get:**
A `computer`, or `shoes`, or `backpack` or `bycicle` or `furniture` or `office` or `apartement` or `flight` or `train ticket` or `ship ticket` or `supermarket item` or whatever...

Some people are _social ninjas_ and they make their way through. My parents are not. My friends/family/coworkers/... pretty much anyone i know do not have survival skills to get through the days and weeks without money. They would end up as bumps on the street.

So yes - nice philosophical argument - that **money** !== **ressources**, but as a matter of every day life fact, it's the only way through which complete strangers are willing to give you any items you need to live and survive.

### What problem is money meant to solve?

Try again to get any of those items above without money. Sometimes you are lucky and people give it for free - but if you dont have the social skills or kind environment - then you need to offer something you can proivde.
**For example**:
1. Work for an hour for someone to produce something.
2. Have the luck, that this guy actually can provide you exactly with the item you need.

Basically:
Exchange apples for bananas for clothing for tools for a haircut for a bycicle for a smartphone for...
Find the right supply chain or lets better say "Supply Circle".
Thats a hard problem a computer could solve if all the "supply" and "demand" would be published by EVERYONE in a way so that they could be MATCHED automatically.

Until then we need money, because people dont trust they will get something if they give for free and because there are always narrow minded people who would not give but take as much as they can, because they could.

**Comment by serapath** [comment]
And regarding your linked article, this is pretty much irrelevant, because honestly... this is such a rare case that its basically fucking irrelevant!
Who "the f..." earns **75000USD/year** a year?
We have 7-8 billion people on the planet and most digital nomads do not earn that much, so its pretty much pointless.

**Comment by ninabreznik** [comment]
Tonny Hsieh from Zappos published an interesting book on that topic - Delivering happiness. It is talking about making customers happy, but on the side it actually explores questions of what happiness really is.

He comes up with this: http://deliveringhappiness.com/wp-content/uploads/2010/06/dhb_framework3.png
https://www.youtube.com/watch?v=QDuyqtr3loc

**Comment by nicksellen** [comment]
> happyness in my experience comes from being with others and making them happy

@serapath I understand about the requirement for some money, I was just wondering how it fits into that bit.

> Who "the f..." earns 75000USD/year a year?

It's not as much as it sounds. From a quick google the median salary in the USA is $50k/year, compared to German median of €22k/year, so it's about equivalent to someone in Germany earning around €33k/year. I think it's because the state is paying for less, so they have to pay for more.

@ninabreznik ah yes Kai mentioned that model to me before. Maybe I should read the book. I wonder how those categories came about. The video might even say, but my eyes must close right now.

**Comment by serapath** [comment]
@nicksellen 
If someone earns 75000 USD/year, he is included in THE TOP 1% RICHEST PEOPLE ON THIS PLANET. And yes, it's not "a lot", because there are people who are insanely rich, so if you calculate the AVERAGE salary, it's very easy to get to something like 22k/year or 33k/year, because actually you just need one single person earning a universe of wealth to increase the mean to somethink like that.

Check out
http://www.globalrichlist.com/

**Comment by nicksellen** [comment]
The salary figures I quoted are median salaries, for the reason you mention - it means 50% of the population earn more than that and 50% less. One person earning a huge amount doesn't skew the median.

If you look at http://en.wikipedia.org/wiki/Household_income_in_the_United_States#/media/File:Distribution_of_Annual_Household_Income_in_the_United_States_2012.png you can see that about 30% of the USA earn 75000USD/year or more.

**Comment by nicksellen** [comment]
Sorry, all these figures are _household_ income too (although it's not always clear). Household income could mean 50/50 split between partners so 37500USD/year or converting it via German median income and into euros, €16.5k/year each for a couple (equates to this 75000USD/year). Approximately, from what I can tell.

**Comment by serapath** [comment]
I'm not interested in discussing problems of people who earn more than 75k / year

**Comment by nicksellen** [comment]
They are people too :)

... and they often aren't having a nicer time on this planet anyway. Money does not buy freedom or happiness.

I don't think _class warfare_ is productive. The rich are not my enemy.

**Comment by serapath** [comment]
I dont believe in "classes" and even less in something like "class warfare" whatever that would be.
Money can buy freedom and even though it cannot buy happiness, it can create the best possible conditions for it - but yes granted, money alone wont help you to become happy :-)

**Comment by nicksellen** [comment]
> I'm not interested in discussing problems of people who earn more than 75k / year

I interpreted this bit as _class warfare_; people who earn over 75k/year are the _other class_ and their problems don't count - what do you mean by it?

**Comment by serapath** [comment]
Of course their problems count and are interesting to themselves. Lots of people have lots of problems, but if it happens to be, that some people have 75k/year income, then they got rid of one major "problem" which plagues the vast majority of people on earth.
It's kind of a luxury to be able to tackle the problems you are left with, once you have that income.
So I'd actually like to ponder about how to enable/empower people to actually be able to confront those problems you might be face with, once you have a 75k+ income :-)

I was a bit shocked when you mentioned "warfare", because I dont see what and who against what or whom and what that could potentially help. The way humans currently organize their economy enables a lot of benefits never seen before, but it also has some shortcomings, which I believe should be solved in an entrepreneurial way :-) **_=> Inovate to disrupt banking for the greater good of humankind**_ :-)
